### PR TITLE
Update Group API limit + Update Engage API issues

### PR DESCRIPTION
**Pull request by kurbycchua**
1. Update Group API limit from 200 to 2000
2. Update double descriptions in Engage API
3. Update example for engage batch-update and indicate that all commands are supported
4. Update text for group batch-update to indicate that all commands are supported

### CODE REVIEW

File: openapi/src/ingestion.openapi.yaml [hunk 1]
```diff
@@ -1066,7 +1062,11 @@ paths:
         Send a batch of group profile updates. Instead of sending a single JSON object
         as the data query parameter, send a JSON list of objects as the data
         parameter of an application/x-www-form-urlencoded POST or GET request
-        body
+        body.
+
```
Comment (mavlee) [inline openapi/src/ingestion.openapi.yaml:1066]: nit extra line

File: reference/Ingestion API/user-profiles/user-profile-union.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
 ---
 title: "Union To List Property"
 slug: "user-profile-union"
-excerpt: ""
+excerpt: 'Adds the specified values to a list property on a user profile and ensures that those values only appear once. The profile is created if it does not exist.'
```
Comment (mavlee) [inline reference/Ingestion API/user-profiles/user-profile-union.md]: double quote for consistency

File: reference/Ingestion API/user-profiles/delete-profile.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 ---
 title: "Delete Profile"
 slug: "delete-profile"
-excerpt: ""
+excerpt: "Permanently delete the profile from Mixpanel, along with all of its properties. The $delete object value is ignored - the profile is determined by the $distinct_id from the request itself.
```
Comment (mavlee) [inline reference/Ingestion API/user-profiles/delete-profile.md:4]: what does putting it in excerpt do?

File: openapi/src/ingestion.openapi.yaml [hunk 4]
```diff
@@ -404,18 +404,6 @@ paths:
         If the profile does not exist, it creates it with these properties. If
         it does exist, it sets the properties to these values, overwriting
         existing values.
-        [block:callout]
```
Comment (mavlee) [inline openapi/src/ingestion.openapi.yaml:407]: why was this removed?

File: openapi/src/ingestion.openapi.yaml (hunk 1 above)
Comment (kurbycchua) [reply openapi/src/ingestion.openapi.yaml:1066]: not quite sure what you mean by this @mavlee but guessing you mean the extra line at 1069 and not the 2 lines 1066 and 1067 which I deliberately added to create an extra space otherwise the rendering in readme didn't put an extra space somehow when I tried before.

File: reference/Ingestion API/user-profiles/user-profile-union.md (hunk 2 above)
Comment (kurbycchua) [reply reference/Ingestion API/user-profiles/user-profile-union.md]: updated

File: reference/Ingestion API/user-profiles/delete-profile.md (hunk 3 above)
Comment (kurbycchua) [reply reference/Ingestion API/user-profiles/delete-profile.md:4]: not sure but like you said for consistency I am doing it since the rest has it. I find it does not do anything, only text outside that block actually shows up in the dev docs which I have removed since it's creating double betweey yaml file and the text from md file

File: openapi/src/ingestion.openapi.yaml (hunk 4 above)
Comment (kurbycchua) [reply openapi/src/ingestion.openapi.yaml:407]: because it's the one causing the rendering issue, see here: https://developer.mixpanel.com/reference/profile-set

**Review by mavlee — APPROVED** [review]
